src/services/change-requests.api.ts

```diff
@@ -3,18 +3,29 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+import { CR_Type } from '@prisma/client';
 import axios from 'axios';
+import { NewStandardChangeRequestPayload } from 'utils';
 import { ChangeRequest } from 'utils';
+import { NewActivationChangeRequestPayload, NewStageRequestChangeRequestPayload } from 'utils/src';
 import { apiUrls } from '../shared/urls';
 import { changeRequestTransformer } from './transformers/change-requests.transformers';
 
 /**
  * Fetches all change requests.
  */
-export const getAllChangeRequests = () => {
-  return axios.get<ChangeRequest[]>(apiUrls.changeRequests(), {
+export const getAllChangeRequests = (onSuccess?: (value: any) => void) => {
```

Comment:
just curious to understand why. I'd be inclined to open a tech debt ticket to reexamine this onSuccess pattern and understand if it's needed or if there's a better way to handle it, but that can definitely be left for after MVP launch

Reply:
Let's examine after MVP; we definitely need some sort of way to pass functions to the async functions.